src/graphql/resolvers/projectAssignmentResolver.ts

```diff
@@ -0,0 +1,36 @@
+import {
+	createProjectAssignment,
+	getProjectAssignment,
+	updateProjectAssignment,
+} from "../../helpers/projectAssignmentService";
+import { Arg, Int, Mutation, Query, Resolver } from "type-graphql";
+import {
+	CreateProjectAssignmentInputModel,
+	ProjectAssignmentModel,
+	UpdateProjectAssignmentInputModel,
+} from "../models/projectAssignmentModel";
+
+@Resolver()
+export class ProjectAssignmentResolver {
+	@Query((_returns) => [ProjectAssignmentModel])
+	async projectAssignment() {
+		return getProjectAssignment();
```

Comment:
same comment as the departments query. if returning multiple then make it plural.